src/dispatch/static/dispatch/src/router/index.js

```diff
@@ -138,7 +152,7 @@ router.beforeEach((to, from, next) => {
     if (authProviderSlug === "dispatch-auth-provider-pkce") {
       loginwithPKCE(to, from, next)
     } else {
-      next()
```

Comment:
I think we need to provide a `dispatch-auth-provider-basic` slug. There are users who don't wish to enabled any kind of auth. That said I'm okay making `dispatch-auth-provider-basic` the default here: https://github.com/Netflix/dispatch/blob/develop/src/dispatch/config.py#L75

Reply:
I see what you are saying, I added to the config "dispatch-auth-provider-basic". I did not know this app is usable without any authentication.